mailpoet/mailpoet.php

```diff
@@ -97,8 +97,14 @@ function mailpoet_core_dependency_notice() {
 
 // Display PHP version error notice
 function mailpoet_php_version_notice() {
-  $noticeP1 = __('MailPoet requires PHP version 7.4 or newer (8.1 recommended). You are running version [version].', 'mailpoet');
-  $noticeP1 = str_replace('[version]', phpversion(), $noticeP1);
+  $noticeP1 = sprintf(
+    // translators: %1$s is the plugin name (MailPoet or MailPoet Premium), %2$s, %3$s, and %4$s are PHP version (e.g. "8.1.30")
+    __('%1$s requires PHP version %2$s or newer (%3$s recommended). You are running version %4$s.', 'mailpoet'),
```

Comment:
Not sure if the "MailPoet" "MailPoet Premium" replacement is of value, since this string is then just double in two different projects/text-domains (mailpoet and mailpoet-premium), but no objections. The other replacements are 👍

Reply:
Yeah, I originally wanted to use the `mailpoet` text domain also in premium, but I wasn't sure if that has any implications. But at least this way, Transifex can use the same translation for both strings, even if they are in different projects, so in practice, they should be translated automatically.